src/iotjs_binding.h

```diff
@@ -224,4 +224,22 @@ jerry_value_t iotjs_jhelper_eval(const char* name, size_t name_len,
 
 jerry_value_t vm_exec_stop_callback(void* user_p);
 
+#ifdef __GNUC__
```

Comment:
@daeyeon Does clang support `__cleanup__`? If not, it will break build support for macOS.

Reply:
@daeyeon Currently macOS build is ok (Travis `host-darwin` build passes.) because clang defines `__GNUC__` for GCC compatibility. But it does not guarantee it will work correctly. The `__cleanup__` may be ignored. We need to check it.